src/core/loader/cia.cpp

```diff
@@ -0,0 +1,162 @@
+// Copyright 2017 Citra Emulator Project
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+
+#include <cstring>
+#include <memory>
+#include <string>
+#include "common/common_types.h"
+#include "common/file_util.h"
+#include "common/logging/log.h"
+#include "common/symbols.h"
+#include "core/hle/kernel/process.h"
+#include "core/hle/kernel/resource_limit.h"
+#include "core/loader/cia.h"
+#include "core/memory.h"
+#include "cia.h"
+namespace Loader {
```

Comment:
I'd have expected some empty lines up to this point - please check with the other files or use clang-format to tell you (which might not know everything though)